### PR TITLE
fix: multiple fixes on Panduit odf and cassettes

### PR DESCRIPTION
**Pull request by sjurtf**
These Panduit 12x duplex LC MPO Cassettes have 2x MPO 12 fiber rear ports, not a single MPO 24 fiber.
All positions are duplex.

- https://www.panduit.com/en/products/fiber-optic-systems/fiber-optic-panels-cassettes-enclosures/fiber-optic-cassettes/fc2xo-24-10as.html
- https://www.panduit.com/en/products/fiber-optic-systems/fiber-optic-panels-cassettes-enclosures/fiber-optic-cassettes/fc2xo-24-10u.html
- https://www.panduit.com/en/products/fiber-optic-systems/fiber-optic-panels-cassettes-enclosures/fiber-optic-cassettes/fc29n-24-10as.html
- https://www.panduit.com/en/products/fiber-optic-systems/fiber-optic-panels-cassettes-enclosures/fiber-optic-cassettes/fc29n-24-10u.html

### CODE REVIEW

**Review by DanSheps — CHANGES_REQUESTED** [review]
Hello,

I think these files actually need a complete re-write.

With the addition of support for duplex fiber connections (multi-connection) these should be re-written as 24x1 front ports and 2x12 rear ports.

If you would like to undertake this, I would recommend the following:

Rename the ports 01 to 24
Label them following the following datasheets:
FC29N-24-10AS: https://www.panduit.com/content/dam/panduit/en/products/media/2/52/852/8852/110058852.pdf
FC29N-24-10U: https://www.panduit.com/content/dam/panduit/en/products/media/0/80/880/8880/110058880.pdf
FC2XO-24-10AS: https://www.panduit.com/content/dam/panduit/en/products/media/7/87/987/5987/98235987.pdf (Same fiber pairing as Method A SM fiber cassette)
FC2XO-24-10U: https://www.panduit.com/en/products/fiber-optic-systems/fiber-optic-panels-cassettes-enclosures/fiber-optic-cassettes/fc2xo2410u.html (Same fiber pairing as UIniversal SM fiber cassette)

TLDR;

Universal:

(Label): (Name [Strand])
01-A: 12
01-B: 01
02-A: 11
02-B: 02
03-A: 10
03-B: 03
04-A: 09
04-B: 04
05-A: 08
05-B: 05
06-A: 07
06-B: 06
07-A: 24
07-B: 13
08-A: 23
08-B: 14
09-A: 22
09-B: 15
10-A: 21
10-B: 16
11-A: 20
11-B: 17
12-A: 19
12-B: 18

For Method A:
(Label): (Name [Strand])
01-A: 01
01-B: 02
02-A: 03
02-B: 04
03-A: 05
03-B: 06
04-A: 07
04-B: 08
05-A: 09
05-B: 10
06-A: 11
06-B: 12
07-A: 13
07-B: 14
08-A: 15
08-B: 16
09-A: 17
09-B: 18
10-A: 19
10-B: 20
11-A: 21
11-B: 22
12-A: 23
12-B: 24

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 1]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
 model: Opticom MPO-LC Fiber Cassette OS2, 24 Fiber, Universal
 part_number: FC29N-24-10U
 front-ports:
-  - name: '01'
+  - name: '{module}:01'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 01-A
     rear_port_position: 1
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:10]:
```suggestion
    rear_port_position: 12
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 2]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '02'
+  - name: '{module}:02'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 01-B
+    rear_port_position: 12
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:15]:
```suggestion
    rear_port_position: 1
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 3]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:03'
+    type: lc
+    rear_port: '{module}:1'
+    label: 02-A
     rear_port_position: 2
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:20]:
```suggestion
    rear_port_position: 11
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 4]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 02-B
+    rear_port_position: 11
+  - name: '{module}:05'
+    type: lc
+    rear_port: '{module}:1'
+    label: 03-A
     rear_port_position: 3
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:30]:
```suggestion
    rear_port_position: 10
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 5]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+    rear_port: '{module}:1'
+    label: 02-A
     rear_port_position: 2
-  - name: '03'
+  - name: '{module}:04'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 02-B
+    rear_port_position: 11
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:25]:
```suggestion
    rear_port_position: 2
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 6]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+    rear_port: '{module}:1'
+    label: 03-A
     rear_port_position: 3
-  - name: '04'
+  - name: '{module}:06'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 03-B
+    rear_port_position: 10
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:35]:
```suggestion
    rear_port_position: 3
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 7]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:07'
+    type: lc
+    rear_port: '{module}:1'
+    label: 04-A
     rear_port_position: 4
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:40]:
```suggestion
    rear_port_position: 9
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 8]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '05'
+  - name: '{module}:08'
+    type: lc
+    rear_port: '{module}:1'
+    label: 04-B
+    rear_port_position: 9
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:45]: 4

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 9]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:09'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 05-A
     rear_port_position: 5
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:50]:
```suggestion
    rear_port_position: 8
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 10]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '06'
+  - name: '{module}:10'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 05-B
+    rear_port_position: 8
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:55]:
```suggestion
    rear_port_position: 5
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 11]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:11'
+    type: lc
+    rear_port: '{module}:1'
+    label: 06-A
     rear_port_position: 6
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:60]:
```suggestion
    rear_port_position: 7
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 12]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '07'
+  - name: '{module}:12'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 06-B
     rear_port_position: 7
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:65]:
```suggestion
    rear_port_position: 6
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 13]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '08'
+  - name: '{module}:13'
     type: lc
-    rear_port: '01'
-    rear_port_position: 8
-  - name: '09'
+    rear_port: '{module}:2'
+    label: 07-A
+    rear_port_position: 1
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:70]:
```suggestion
    rear_port_position: 12
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 14]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:14'
     type: lc
-    rear_port: '01'
-    rear_port_position: 9
-  - name: '10'
+    rear_port: '{module}:2'
+    label: 07-B
+    rear_port_position: 12
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:75]:
```suggestion
    rear_port_position: 1
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 15]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:15'
     type: lc
-    rear_port: '01'
-    rear_port_position: 10
-  - name: '11'
+    rear_port: '{module}:2'
+    label: 08-A
+    rear_port_position: 2
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:80]:
```suggestion
    rear_port_position: 11
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 16]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:16'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:2'
+    label: 08-B
     rear_port_position: 11
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:85]:
```suggestion
    rear_port_position: 2
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 17]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '12'
+  - name: '{module}:17'
     type: lc
-    rear_port: '01'
-    rear_port_position: 12
+    rear_port: '{module}:2'
+    label: 09-A
+    rear_port_position: 3
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:90]:
```suggestion
    rear_port_position: 10
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 18]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:18'
+    type: lc
+    rear_port: '{module}:2'
+    label: 09-B
+    rear_port_position: 10
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:95]:
```suggestion
    rear_port_position: 3
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 19]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:19'
+    type: lc
+    rear_port: '{module}:2'
+    label: 10-A
+    rear_port_position: 4
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:100]:
```suggestion
    rear_port_position: 9
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 20]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:20'
+    type: lc
+    rear_port: '{module}:2'
+    label: 10-B
+    rear_port_position: 9
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:105]:
```suggestion
    rear_port_position: 4
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 21]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:21'
+    type: lc
+    rear_port: '{module}:2'
+    label: 11-A
+    rear_port_position: 5
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:110]:
```suggestion
    rear_port_position: 8
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 22]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:22'
+    type: lc
+    rear_port: '{module}:2'
+    label: 11-B
+    rear_port_position: 8
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:115]:
```suggestion
    rear_port_position: 5
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 23]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:23'
+    type: lc
+    rear_port: '{module}:2'
+    label: 12-A
+    rear_port_position: 6
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:120]:
```suggestion
    rear_port_position: 7
```

File: module-types/Panduit/FC29N-24-10U.yaml [hunk 24]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:24'
+    type: lc
+    rear_port: '{module}:2'
+    label: 12-B
+    rear_port_position: 7
```
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:125]:
```suggestion
    rear_port_position: 6
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 25]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
 model: Opticom MPO-LC Fiber Cassette OM3, 24 Fiber, Universal
 part_number: FC2XO-24-10U
 front-ports:
-  - name: '01'
+  - name: '{module}:01'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 01-A
     rear_port_position: 1
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:10]:
```suggestion
    rear_port_position: 12
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 26]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '02'
+  - name: '{module}:02'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 01-B
+    rear_port_position: 12
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:15]:
```suggestion
    rear_port_position: 1
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 27]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:03'
+    type: lc
+    rear_port: '{module}:1'
+    label: 02-A
     rear_port_position: 2
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:20]:
```suggestion
    rear_port_position: 11
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 28]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '03'
+  - name: '{module}:04'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 02-B
+    rear_port_position: 11
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:25]:
```suggestion
    rear_port_position: 2
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 29]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:05'
+    type: lc
+    rear_port: '{module}:1'
+    label: 03-A
     rear_port_position: 3
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:30]:
```suggestion
    rear_port_position: 10
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 30]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '04'
+  - name: '{module}:06'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 03-B
+    rear_port_position: 10
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:35]:
```suggestion
    rear_port_position: 3
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 31]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:07'
+    type: lc
+    rear_port: '{module}:1'
+    label: 04-A
     rear_port_position: 4
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:40]:
```suggestion
    rear_port_position: 9
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 32]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '05'
+  - name: '{module}:08'
+    type: lc
+    rear_port: '{module}:1'
+    label: 04-B
+    rear_port_position: 9
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:45]:
```suggestion
    rear_port_position: 4
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 33]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:09'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 05-A
     rear_port_position: 5
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:50]:
```suggestion
    rear_port_position: 8
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 34]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '06'
+  - name: '{module}:10'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 05-B
+    rear_port_position: 8
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:55]:
```suggestion
    rear_port_position: 5
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 35]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:11'
+    type: lc
+    rear_port: '{module}:1'
+    label: 06-A
     rear_port_position: 6
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:60]:
```suggestion
    rear_port_position: 7
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 36]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '07'
+  - name: '{module}:12'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:1'
+    label: 06-B
     rear_port_position: 7
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:65]:
```suggestion
    rear_port_position: 6
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 37]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '08'
+  - name: '{module}:13'
     type: lc
-    rear_port: '01'
-    rear_port_position: 8
-  - name: '09'
+    rear_port: '{module}:2'
+    label: 07-A
+    rear_port_position: 1
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:70]:
```suggestion
    rear_port_position: 12
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 38]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:14'
     type: lc
-    rear_port: '01'
-    rear_port_position: 9
-  - name: '10'
+    rear_port: '{module}:2'
+    label: 07-B
+    rear_port_position: 12
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:75]:
```suggestion
    rear_port_position: 1
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 39]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:15'
     type: lc
-    rear_port: '01'
-    rear_port_position: 10
-  - name: '11'
+    rear_port: '{module}:2'
+    label: 08-A
+    rear_port_position: 2
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:80]:
```suggestion
    rear_port_position: 11
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 40]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:16'
     type: lc
-    rear_port: '01'
+    rear_port: '{module}:2'
+    label: 08-B
     rear_port_position: 11
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:85]:
```suggestion
    rear_port_position: 2
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 41]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
-  - name: '12'
+  - name: '{module}:17'
     type: lc
-    rear_port: '01'
-    rear_port_position: 12
+    rear_port: '{module}:2'
+    label: 09-A
+    rear_port_position: 3
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:90]:
```suggestion
    rear_port_position: 10
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 42]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:18'
+    type: lc
+    rear_port: '{module}:2'
+    label: 09-B
+    rear_port_position: 10
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:95]:
```suggestion
    rear_port_position: 3
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 43]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:19'
+    type: lc
+    rear_port: '{module}:2'
+    label: 10-A
+    rear_port_position: 4
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:100]:
```suggestion
    rear_port_position: 9
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 44]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:20'
+    type: lc
+    rear_port: '{module}:2'
+    label: 10-B
+    rear_port_position: 9
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:105]:
```suggestion
    rear_port_position: 4
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 45]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:21'
+    type: lc
+    rear_port: '{module}:2'
+    label: 11-A
+    rear_port_position: 5
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:110]:
```suggestion
    rear_port_position: 8
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 46]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:22'
+    type: lc
+    rear_port: '{module}:2'
+    label: 11-B
+    rear_port_position: 8
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:115]:
```suggestion
    rear_port_position: 5
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 47]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:23'
+    type: lc
+    rear_port: '{module}:2'
+    label: 12-A
+    rear_port_position: 6
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:120]:
```suggestion
    rear_port_position: 7
```

File: module-types/Panduit/FC2XO-24-10U.yaml [hunk 48]
```diff
@@ -3,55 +3,130 @@ manufacturer: Panduit
...
+  - name: '{module}:24'
+    type: lc
+    rear_port: '{module}:2'
+    label: 12-B
+    rear_port_position: 7
```
Comment (DanSheps) [inline module-types/Panduit/FC2XO-24-10U.yaml:125]:
```suggestion
    rear_port_position: 6
```

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: module-types/Panduit/FC29N-24-10U.yaml (hunk 8 above)
Comment (DanSheps) [inline module-types/Panduit/FC29N-24-10U.yaml:45]:
```suggestion
    rear_port_position: 4
```
